babel.config.js

```diff
@@ -2,7 +2,12 @@ module.exports = function (api) {
   api.cache(true);
 
   const presets = [
-    '@babel/preset-react',
+    [
```

Comment:
From [React 17 docs](https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html#manual-babel-setup)